frontend/src/pages/team/create.vue

```diff
@@ -151,6 +155,15 @@ export default {
             return this.billingEnabled &&
                    !this.user.admin &&
                    this.input.teamType && this.input.teamType.properties?.billing?.requireContact
+        },
+        isSelectionTrial () {
+            // A team trial can be offered if:
+            // 1. User has no other teams
+            return this.teams.length === 0 &&
+            // 2. User is less than a week old
+                (Date.now() - (new Date(this.user.createdAt)).getTime()) < 1000 * 60 * 60 * 24 * 7 &&
```

Comment:
worth centralising this logic somewhere? Feels like something that would be easily overlooked if we change trial lengths in the future?